File: mycuration/src/main/java/com/phicdy/mycuration/rss/Curation.java
package com.phicdy.mycuration.rss;



public class Curation {
	private final int id;
	private final String name;

	public static final String TABLE_NAME = "curations";
	public static final String NAME = "name";
	public static final String ID = "_id";

	public static final String CREATE_TABLE_SQL =
			"create table " + TABLE_NAME + "(" +
					ID + " integer primary key autoincrement,"+
					NAME + " text)";

	public Curation(int id, String name) {
		this.id = id;
		this.name = name;
	}
	
	public Integer getId() {
		return id;
	}
	
	public String getName() {
		if (name == null) return "";
		return name;
	}
}

File: mycuration/src/main/java/com/phicdy/mycuration/data/rss/Curation.kt
package com.phicdy.mycuration.data.rss


data class Curation(
        val id: Int,
        val name: String
) {

    companion object {

        const val TABLE_NAME = "curations"
        const val NAME = "name"
        const val ID = "_id"

        const val CREATE_TABLE_SQL = "create table " + TABLE_NAME + "(" +
                ID + " integer primary key autoincrement," +
                NAME + " text)"
    }
}
